packages/subnav/partials/CtaLinks/index.js

```diff
@@ -41,6 +40,8 @@ function CtaLinks(props) {
             theme={{
               brand: product,
               variant: isLastButton ? 'primary' : 'secondary',
+              background: theme,
+              ...link.theme /* allow theme overrides via the ctaLinks array */,
```

Comment:
Added this in to remove a related override in cloud.hashicorp.com used to emulate a `secondary` button. Consumers can now add a `link.theme` attribute to set button themes, if desired.